test.properties

```diff
@@ -6752,16 +6752,7 @@
         )
 
     um.playwright.projects=\
-        account-admin-web.main,\
-        address-web.main,\
-        commerce-account-web.main,\
-        portal-default-permissions-web.main,\
-        portal-user-locale-options-web.main,\
-        product-navigation-user-personal-bar-web.main,\
-        roles-admin-web.main,\
-        roles-selector-web.main,\
-        users-admin-web.main,\
-        users-admin-web.permissions
+        users-admin-web.email
```

Comment:
this can be reverted (or better, you need to add this new project with all the others)